Projects/templates/Projects/profile.html

```diff
@@ -44,7 +44,11 @@
                                 </div>
                                 <div class="card-text">
                                     {% if request.user == pr.from_user %}
```

Comment:
you should have just modified this if ` {% if request.user == pr.from_user or request.user.profile.role == 'mentor' %}`

Reply:
that's okay too but this is just a new check rather adding whole new delete icon  :smile:

Reply:
see, all mentors are must to be staff but not vice versa

Reply:
I tried above but it didn't work. It was giving some error. I tried many different ways but couldn't do it with only one if statement

Reply:
I have updated this to be just a single if else, it should be working :smile: